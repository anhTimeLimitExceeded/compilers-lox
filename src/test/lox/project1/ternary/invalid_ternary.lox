print "true" ? "missing colon";
print "true" ? "missing" : "semicolon"
