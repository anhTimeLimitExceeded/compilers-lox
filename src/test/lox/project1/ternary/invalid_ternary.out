[line 1] Error at '?': Expecting ':' for ternary expression
[line 3] Error at end: Expect ';' after value.
